src/utils/transition-manager.js

```diff
@@ -0,0 +1,180 @@
+/* global requestAnimationFrame, cancelAnimationFrame */
```

Comment:
Assuming that this file and the viewport utils below are copy-pasta from deck.gl - do we have tests for them?